crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -266,6 +267,33 @@ fn reduce_mean_update_outputs(node: &mut Node) {
     }
 }
 
+/// Update the output tensor dimension
+fn squeeze_update_output(node: &mut Node) {
+    if let Some(Data::Int64s(axes)) = &node.inputs[1].value {
+        if axes.len() != 1 {
+            panic!("Squeeze: Only one axis should be specified for squeezing.");
+        }
+    } else {
+        panic!("Squeeze: Axes input must be an integer list.");
+    };
```

Comment:
If we're checking for axes in the attributes in `squeeze_config` to support another opset we need to check here too to make sure the output is properly adjusted.

See the `unsqueeze_update_output` function which is doing something similar (though it captures the axes to support unsqueeze on multiple axes).

Reply:
Sounds good, should be a little closer to what you had in mind now!